kubectl-plugin/kubectl-waiops

```diff
@@ -14,13 +14,15 @@ white=$(tput setaf 7)
 bold=$(tput bold)
 normal=$(tput sgr0) #reset color/bolding
 
-# Get installation namespace (inferred from installation instance)
-PROJECT_NAMESPACE=$(oc get installations.orchestrator.aiops.ibm.com -A --no-headers | while read a b c; do echo "$a"; done;)
+# Get installation namespace (gathered from existing installation instance on the cluster)
+INSTALLATION_NAME=$(oc get installations.orchestrator.aiops.ibm.com -A --no-headers | while read a b c; do echo "$b"; done;)
+INSTALLATION_NAMESPACE=$(oc get installations.orchestrator.aiops.ibm.com -A --no-headers | while read a b c; do echo "$a"; done;)
```

Comment:
@taylormgeorge91 going back to our discussion on Slack earlier about the placement of this, this actually needs to be outside the arguments because it's used for both `oc waiops status` and `oc waiops upgrade-status`. Will need to keep it where it is currently located.